src/store/plugins/names.js

```diff
@@ -85,7 +85,7 @@ export default (store) => {
                         { root: true },
                       );
                     return {
-                      ...(revoked || oldName.revoked ? { revoked: true } : {}),
+                      ...(name.revoke ? { revoked: true } : {}),
```

Comment:
So, `revoke` will be renamed to `revoked` only if `oldName` exist? I can't get why an `oldName` check is needed at all. Also, does middleware return **revoked** names in the list of **active** names?? Is there an issue about this?